consent-ui/src/app/common/auth.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import { PsuAuthService } from '../api-auth/api/psuAuth.service'
+import * as uuid from 'uuid';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  constructor(
+    private http: HttpClient,
+    private psuAuthService: PsuAuthService
+  ) {  }
+
+  public userLogin(formData) {
```

Comment:
use models generated by the API `PsuAuthBody` instead of any and give `formData` a meaningful name